packages/create-cloudflare/src/frameworks/angular/index.ts

```diff
@@ -44,10 +44,10 @@ const config: FrameworkConfig = {
 	packageScripts: {
 		process:
 			"node ./tools/copy-worker-files.mjs && node ./tools/copy-client-files.mjs && node ./tools/bundle.mjs",
-		prestart: "npm run build:ssr && npm run process",
+		prestart: `${npm} run build:ssr && npm run process`,
```

Comment:
Oops!

Should be:

```
`${npm} run build:ssr && ${npm} run process`
```

Reply:
Fixed in https://github.com/cloudflare/workers-sdk/pull/3486/files#diff-eafbea9c833fea865a227b2b86c2948493ac2c97f2c6f275572af5976c3628b9R47